Dockerfile

```
@@ -1,9 +1,10 @@
 FROM java:openjdk-8-jre-alpine
 
-RUN apk update && apk add --no-cache unzip
-RUN adduser -D -h /home/ircbot -u 1013 ircbot
-ADD target/ircbot-2.0-SNAPSHOT-bin.zip /usr/local/bin/ircbot.zip
-RUN cd /usr/local/bin; unzip ircbot.zip
+RUN apk update && apk add --no-cache unzip=6.0-25ubuntu1 && \
```

```suggestion
RUN apk add --no-cache unzip && \
```
When using alpine linux:
- The `apk update` is not needed (it's implied by the flag `--no-cache` of `apk add <...>`)
- You are providing a version of unzip that seems.. Ubuntu-y :D

=> I wonder if unzip is really required: 
- we might want to use the [`ADD`](https://docs.docker.com/engine/reference/builder/#add) instruction that is able to unarchive (but I 'm not sure if zip archives are required)
- And/or we could avoid zipping the jar file during the `make <..>` steps
- And/or using a multi stage image to ensure that `unzip` is not present on the production image

---

Made a jar instead of a zip, and removed unzip :)